src/server/pps/server/gc.go

```diff
@@ -57,16 +57,17 @@ func (a *apiServer) runGC(ctx context.Context, pipelineInfo *ppsclient.PipelineI
 			continue
 		}
 		for _, jobID := range jobIDs.Jobs {
-			zero := int64(0)
-			falseVal := false
-			go func(jobID string) {
-				if err := a.kubeClient.Extensions().Jobs(a.namespace).Delete(jobID, &api.DeleteOptions{
-					GracePeriodSeconds: &zero,
-					OrphanDependents:   &falseVal,
-				}); err != nil {
-					// TODO: if the error indicates that the job has already been
-					// deleted, just proceed to the next step
-					log.Errorf("error deleting kubernetes job %s: %s", jobID, err)
+			jobID := jobID
+			go func() {
+				jobInfo, err := client.InspectJob(ctx, &ppsclient.InspectJobRequest{
+					Job: &ppsclient.Job{ID: jobID},
+				})
+				if err != nil {
+					log.Errorf("error deleting job: %s", err)
+					return
+				}
+				if err := a.deleteJob(ctx, jobInfo); err != nil {
+					log.Errorf("error deleting job: %s", err)
```

Comment:
Yea --- those lines had to swap out 'protolion' for the new logger 'log' and that was part of a merge conflict fix when merging in master.